controllers/news.controllers.js

```diff
@@ -1,10 +1,10 @@
+const comments = require("../db/data/test-data/comments");
```

Comment:
Why are you requiring the test data here?

Reply:
I don't know where it came from lol > I have it removed, now, Karla.